benchmark.go

```diff
@@ -65,11 +65,18 @@ type Summary struct {
 	RPS                float64 `json:"rps"`
 }
 
+// StreamSummary stores the stream messages send and received summary
+type StreamSummary struct {
+	TotalStreamMessagesSent     int `json:"totalStreamMessagesSent,omitempty"`
+	TotalStreamMessagesReceived int `json:"totalStreamMessagesReceived,omitempty"`
+}
+
 // BenchmarkOutput stores benchmark settings and results for JSON output
 type BenchmarkOutput struct {
-	Parameters Parameters        `json:"benchmarkParameters"`
-	Latencies  map[string]string `json:"latencies"`
-	Summary    Summary           `json:"summary"`
+	Parameters    Parameters        `json:"benchmarkParameters"`
+	Latencies     map[string]string `json:"latencies"`
+	Summary       Summary           `json:"summary"`
+	StreamSummary *StreamSummary    `json:"streamSummary,omitempty"`
```

Comment:
Can you add a comment why you use a pointer here please? I guess it is for the omit empty and to not print anything for unary request

Reply:
Yes, it is omitted in unary benchmark. Added a comment on the field